proto/context/src/main/proto/spine/validation/field_views.proto

```diff
@@ -11,17 +11,30 @@ option java_multiple_files = true;
 
 import "spine/protodata/ast.proto";
 
+// Uniquely identifies a message field.
+message FieldId {
+
+    // Name of the containing type.
+    protodata.TypeName type = 1;
+
+    // Short name of the field.
+    protodata.FieldName name = 2;
+}
+
 // A view on a field that is marked `(required)`.
 message RequiredField {
     option (entity).kind = PROJECTION;
 
     FieldId id = 1;
 
-    // Message appearing when this required field is not set.
+    // The error message template.
     string error_message = 2;
 
     // `true` if the field is marked with `(required) = true`, `false` otherwise.
-    bool required = 3;
+    bool enabled = 3;
```

Comment:
The field was called `required` because it's written like that in the consumer code.

"Enabled" is about something else. "Whether the validation is enabled or not?"

Reply:
It is a bit weird both ways. 

I suggest not create `RequiredField` view for non-required fields at all. The same with the rest of such views. We do nothing when such a flag is `false` because `RequiredField` with `required = false` is `NonRequiredField`.

Having this `bool something = false` flag in `SomethingField` means this `something field` is not `something`.